### PR TITLE
Minor changes in blacklisting nouveau

### PR DESCRIPTION
**Pull request by fktkrt**
I had some problems earlier with the the original steps -even in other projects-, what solved my problem was using `options nouveau modeset=0`

This is disabling only the nouveau driver while for example nomodset disables more modules.
Going this way I was able to install drivers on several other distributions as well. 

I know this is a centos setup, but this might help someone someday. :)

### CODE REVIEW

**Comment by sethgoldin** [comment]
Sorry, not quite sure I understand. Are you saying that `# echo 'blacklist nouveau' >> /etc/modprobe.d/blacklist.conf` alone didn't work for you on CentOS 8?

**Comment by fktkrt** [comment]
No, I had problems with it on the previous major version, 7.7-1908, also on Ubuntus, different projects though.
Disabling it via `options nouveau modeset=0` is the "[official ](https://docs.nvidia.com/pdf/CUDA_Installation_Guide_Linux.pdf)NVIDIA recommendation" too, I think it is just more foolproof this way.

**Comment by sethgoldin** [comment]
Got it. I can't test this at the moment, but this seems like a reasonable pull request, especially given the linked NVIDIA documentation. Thanks for this.

You may be interested to know that the only reason these CentOS 8 instructions are referring to the upstream `.run` file from NVIDIA is because the third-party repositories like ELRepo and RPM Fusion still haven't packaged the NVIDIA drivers for CentOS 8. In the previous CentOS 7 instructions, I just used ELRepo:

- https://github.com/sethgoldin/install-davinci-resolve-centos/pull/7/files#diff-04c6e90faac2675aa89e2176d2eec7d8L50
- https://github.com/sethgoldin/install-davinci-resolve-centos/pull/7/files#diff-04c6e90faac2675aa89e2176d2eec7d8L58

If you're still running CentOS 7, you may just want to install ELRepo and use that. Though [the ELRepo page for the package](https://elrepo.org/tiki/kmod-nvidia) refers to some older versions, I believe that they still regularly package the newest drivers, and `bash` autocomplete should direct you to the latest version if you just start to type out: `# yum install kmod-nvidia-440.44` and hit `Tab`.

When ELRepo finally packages the drivers for CentOS 8, I'll likely just axe these `.run` file instructions, and replace with ELRepo instead.

I have another question for you before I merge this pull request, though. The NVIDIA documentation, on p. 19 of your linked file, says to create the file in `/etc/modprobe.d`, but your pull request specifies `/usr/lib/modprobe.d/`. Why is that?

File: README.md [hunk 1]
```diff
@@ -48,7 +48,11 @@ These particular notes were originally worked out from an installation to an HP
 		```# chmod +x NVIDIA-Linux-x86_64-440.44.run```
 	1. Blacklist the nouveau module:
 		
-		```# echo 'blacklist nouveau' >> /etc/modprobe.d/blacklist.conf```
+		Create a file at `/usr/lib/modprobe.d/blacklist-nouveau.conf` with the following contents:
```
Comment (sethgoldin) [inline README.md]:
```suggestion
 		Create a file at `/etc/modprobe.d/blacklist-nouveau.conf` with the following contents:
```

**Comment by sethgoldin** [comment]
While we're at it, let's also modify that `dracut` line as per NVIDIA's documentation: https://github.com/sethgoldin/install-davinci-resolve-centos/pull/20

**Comment by fktkrt** [comment]
> I have another question for you before I merge this pull request, though. The NVIDIA documentation, on p. 19 of your linked file, says to create the file in `/etc/modprobe.d`, but your pull request specifies `/usr/lib/modprobe.d/`. Why is that?

Yes, sorry, this could happen if you use many distros simultaneously. :)
Fixed it.

**Review by sethgoldin — APPROVED** [review]
Looks good!

**Pull request merged**